### ISSUE TITLE
Error while running ags

### ISSUE THREAD

**Issue by Neil-Tomar** [Aylur/dotfiles#271]
i got an error while running ags on my arch linux and the error says
"

(com.github.Aylur.ags:66699): Gjs-Console-CRITICAL **: 10:50:46.082: supergfxctl -g Graphics mode change error.
\u001b[0;31msupergfxd is not enabled, enable it with `systemctl enable supergfxd\u001b[0m
Error: Zbus error: org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable

(com.github.Aylur.ags:66699): Gjs-Console-CRITICAL **: 10:50:46.101: net.hadess.PowerProfiles is not available

(com.github.Aylur.ags:66699): Gjs-Console-CRITICAL **: 10:50:46.460: asusctl profile -p thread 'main' panicked at asusctl/src/main.rs:62:54:
called `Result::unwrap()` on an `Err` value: MethodError(OwnedErrorName("org.freedesktop.DBus.Error.ServiceUnknown"), Some("The name is not activatable"), Msg { type: Error, serial: 4294967295, sender: UniqueName("org.freedesktop.DBus"), reply-serial: 5, body: Signature("s"), fds: [] })
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
no icon substitute "krfb" for "krfb", fallback: "application-x-executable"
no icon substitute "preferences-desktop-tablet" for "preferences-desktop-tablet", fallback: "application-x-executable"
no icon substitute "start-here-kde-plasma" for "start-here-kde-plasma", fallback: "application-x-executable"
no icon substitute "tools-report-bug" for "tools-report-bug", fallback: "application-x-executable"
no icon substitute "plasma" for "plasma", fallback: "application-x-executable"
no icon substitute "cuttlefish" for "cuttlefish", fallback: "application-x-executable"
no icon substitute "search" for "search", fallback: "application-x-executable"
no icon substitute "krfb" for "krfb", fallback: "application-x-executable"
no icon substitute "preferences-desktop-tablet" for "preferences-desktop-tablet", fallback: "application-x-executable"
no icon substitute "start-here-kde-plasma" for "start-here-kde-plasma", fallback: "application-x-executable"
no icon substitute "tools-report-bug" for "tools-report-bug", fallback: "application-x-executable"
no icon substitute "plasma" for "plasma", fallback: "application-x-executable"
no icon substitute "cuttlefish" for "cuttlefish", fallback: "application-x-executable"
no icon substitute "search" for "search", fallback: "application-x-executable"

(com.github.Aylur.ags:66699): Gjs-Console-WARNING **: 10:50:46.544: Error: expected Pixbuf or string for icon, but got undefined
set icon@resource:///com/github/Aylur/ags/widgets/icon.js:53:26
bind/callback<@resource:///com/github/Aylur/ags/widgets/widget.js:53:30
hook/<@resource:///com/github/Aylur/ags/widgets/widget.js:43:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34
  

(com.github.Aylur.ags:66699): Gjs-CRITICAL **: 10:50:46.544: JS ERROR: Error: Wrong type undefined; string expected
set label@resource:///com/github/Aylur/ags/widgets/label.js:51:9
bind/callback<@resource:///com/github/Aylur/ags/widgets/widget.js:53:30
hook/<@resource:///com/github/Aylur/ags/widgets/widget.js:43:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34

(com.github.Aylur.ags:66699): Gjs-Console-WARNING **: 10:50:46.545: Error: expected Pixbuf or string for icon, but got undefined
set icon@resource:///com/github/Aylur/ags/widgets/icon.js:53:26
bind/callback<@resource:///com/github/Aylur/ags/widgets/widget.js:53:30
hook/<@resource:///com/github/Aylur/ags/widgets/widget.js:43:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34
  

(com.github.Aylur.ags:66699): Gjs-Console-WARNING **: 10:50:46.545: Error: can't assign "org.gnome.Settings-notifications-symbolic" as icon, it is not a file nor a named icon
set icon@resource:///com/github/Aylur/ags/widgets/icon.js:45:30
bind/callback<@resource:///com/github/Aylur/ags/widgets/widget.js:53:30
hook/<@resource:///com/github/Aylur/ags/widgets/widget.js:43:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34
  

(com.github.Aylur.ags:66699): Gjs-CRITICAL **: 10:50:46.547: JS ERROR: TypeError: audio2.microphone.stream is null
MicrophoneMute/</<@file:///tmp/asztal/main.js:2543:7
idle/<@resource:///com/github/Aylur/ags/utils/timeout.js:20:9
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34

no icon substitute "kitty-symbolic" for "kitty-symbolic", fallback: "application-x-executable-symbolic"
no icon substitute "kitty-symbolic" for "kitty-symbolic", fallback: "application-x-executable-symbolic"

(com.github.Aylur.ags:66699): Gjs-Console-WARNING **: 10:50:46.566: Error: expected Pixbuf or string for icon, but got undefined
set icon@resource:///com/github/Aylur/ags/widgets/icon.js:53:26
bind/callback<@resource:///com/github/Aylur/ags/widgets/widget.js:53:30
hook/<@resource:///com/github/Aylur/ags/widgets/widget.js:43:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34
  

(com.github.Aylur.ags:66699): Gjs-CRITICAL **: 10:50:46.566: JS ERROR: Error: Wrong type undefined; string expected
_size@resource:///com/github/Aylur/ags/widgets/icon.js:70:13
set icon@resource:///com/github/Aylur/ags/widgets/icon.js:55:14
bind/callback<@resource:///com/github/Aylur/ags/widgets/widget.js:53:30
hook/<@resource:///com/github/Aylur/ags/widgets/widget.js:43:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34

(com.github.Aylur.ags:66699): Gjs-Console-WARNING **: 10:50:46.566: Error: expected Pixbuf or string for icon, but got undefined
set icon@resource:///com/github/Aylur/ags/widgets/icon.js:53:26
bind/callback<@resource:///com/github/Aylur/ags/widgets/widget.js:53:30
hook/<@resource:///com/github/Aylur/ags/widgets/widget.js:43:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34
  

(com.github.Aylur.ags:66699): Gjs-CRITICAL **: 10:50:46.566: JS ERROR: Error: Wrong type undefined; string expected
_size@resource:///com/github/Aylur/ags/widgets/icon.js:70:13
set icon@resource:///com/github/Aylur/ags/widgets/icon.js:55:14
bind/callback<@resource:///com/github/Aylur/ags/widgets/widget.js:53:30
hook/<@resource:///com/github/Aylur/ags/widgets/widget.js:43:25
_init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:263:34

no icon substitute "kitty-symbolic" for "kitty-symbolic", fallback: "application-x-executable-symbolic"
no icon substitute "kitty-symbolic" for "kitty-symbolic", fallback: "application-x-executable-symbolic"
"

**Comment by aylamar** [comment]
For all of the icons errors, you'll need to add the icons under `assets/` or add a substitute in the `lib/icons.ts` file. As an example, here's what you could add just after line 12 to fix the kitty error:
> 'kitty-symbolic': 'terminal-symbolic',

**Comment by demonspork** [comment]
I also get a lot of these. Is there a way to get a lot more of these icons available or will I need to handle each one independently with an override/manually put in an icon?

**Comment by aylamar** [comment]
@demonspork if you install an icon theme, like [MoreWaita](https://github.com/somepaulo/MoreWaita), those icons will be pulled in if they exist. This is what I opted to use after seeing that Aylur uses it here: https://github.com/Aylur/dotfiles/issues/136

After installing MoreWaita, you might want to restart your system. I ran into something weird where after installing MoreWaita a few icons where full color while the others were monochrome. After a restart, they were all monochrome.
